File: sprint05/JMS/FW/PGS_86491.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(vector<vector<int>> sizes) {
    int max1=0, max2=0;

    for(auto p : sizes){
        max1 = max(max1, min(p[0],p[1]));
        max2 = max(max2, max(p[0],p[1]));
    }
    
    return max1*max2;
}

File: sprint05/KKW/First week/PGS_86491.py
def solution(sizes):
    for i in range(len(sizes)):
        sizes[i].sort()
    width=[]
    length=[]
    for i in range(len(sizes)):
        width.append(sizes[i][0])
        length.append((sizes[i][1]))
    return max(width)*max(length)